Cards/CardContainer.gd

```diff
@@ -321,4 +325,28 @@ func _update_card_positions() -> void:
 				else:
 					card_x += scaled_hover_offset
 		
-		movement_component.state_properties.desired_position.x = card_x
+		# scale card's x value to a range of [-1, 1] from range of [0, viewport width]
+		var viewport_width: float = get_viewport_rect().size.x
+		var card_index_scaled: float = Helpers.convert_from_range(card.global_position.x, 0.0, viewport_width, -1.0, 1.0)
+		
+		# curve the cards in hand by finding the y value on a parabola
+		# y = ax^2 - a where a = max offset and x = card_index scaled to a range of -1 to 1
+		card_y = (pow(card_index_scaled, 2.0) * max_hand_offset_y) - max_hand_offset_y
+		
+		# rotate cards on a cubic curve
+		# y = ax^3 where a = max_rotation and x = value between [-1, 1]
+		var rotation_amount: float = pow(card_index_scaled, 3) * max_rotation
```

Comment:
in the same way that the cards width update based on number of card, does the rotation update based on number of cards ? it probably does based on the card index being cast to -1,1 ?

Reply:
Yeah, so it's using the card's x-position as an input for the y-position and rotation functions. The `card_index_scaled` is an outdated name, I believe I updated this naming to something else in the latest version